charlesinto/iReporter
# Feat create intervention record 162589814

#### What does this PR do?

-  Create new intervention record

#### Description of Task to be completed?
- Implement JWT authentication
- Verify user token
- User should be able to create new report

#### How should this be manually tested?
- Clone the repo - `git clone  git repo address`
- Enter into the project directory - `cd repo folder`
- switch to this branch - `PR branch`
-  Start  the frontend.
- Navigate to `endpoint`
- Open `Postman`
- Grab your token from the application tab in Developers tab
- On postman add the token in headers
- - On `postman` Send a `POST` request to `http://localhost:7000/api/v1/interventions`
- You should receive a successful response

#### Any background context you want to provide?


`

#### What are the relevant pivotal tracker stories?
[#162589814](https://www.pivotaltracker.com/story/show/162589814)

#### Screenshots (if appropriate)

![image](https://user-images.githubusercontent.com/26615085/49827779-6cf35280-fd8a-11e8-9cac-0e0266acefce.png)





#### Questions:
N/A
